src/index.html

```diff
@@ -1,13 +1,282 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Nothing</title>
+    <link rel="stylesheet" href="./styles/style.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      rel="shortcut icon"
+      href="./images/favicon.png"
+      type="image/x-icon"
+    >
+
+    <link href="https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@300;400;500;600;700&family=Space+Mono:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Landing Page</h1>
+    <header class="header page-cover">
+      <div class="menu__top">
+        <a href="#">
+          <img
+            src="images/Logo.svg"
+            alt="Nothing Logo"
+            class="header__logo"
+          >
+        </a>
+
+        <div class="header__icons">
+          <a href="#" class="icon icon--phone"></a>
+          <a href="#sidebar-menu" class="icon icon--sidebar--menu"></a>
+        </div>
+      </div>
+
+      <div class="header__bottom">
+        <h1 class="header__title">Bring joy back to the everyday</h1>
+      </div>
+    </header>
+
+    <aside class="sidebar__menu page-cover" id="sidebar-menu">
+      <div class="sidebar__menu__top menu__top">
+        <a href="#">
+          <img
+            src="images/Logo.svg"
+            alt="Nothing Logo"
+            class="sidebar__menu__logo"
+          >
+        </a>
+        <a href="#" class="icon icon--close"></a>
+      </div>
+
+      <nav class="nav">
+        <ul class="sidebar__menu__links">
+          <li class="nav__item">
+            <a href="#" class="sidebar__menu__link">Products</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="sidebar__menu__link">Store</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="sidebar__menu__link">Login</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#about-us" class="sidebar__menu__link">About us</a>
+          </li>
+        </ul>
+      </nav>
+
+      <a href="tel:+1 234 5555-55-55" class="sidebar__menu__phone">
+        +1 234 5555-55-55
+      </a>
+
+      <a href="tel:+1 234 5555-55-55" class="sidebar__menu__call">
+        call to order
+      </a>
+    </aside>
+
+    <main class="main">
+      <section class="recommended" id="recommended">
+        <h2 class="section-title">Recommended</h2>
+
+        <div class="recommended__products">
+          <article class="product">
+            <img
+              src="./images/products/phone.svg"
+              alt="phone image"
+              class="product__photo"
+            >
```

Comment:
You should increase an image's size when it's hovered [Checklist #7](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Increase%20all%20images%20on%20hover%20(make%20them%20links%20to%20%23%2C%20if%20there%20is%20no%20better%20option))